test/providers/index.test.js

```diff
@@ -99,7 +65,18 @@ describe('CI Providers', () => {
             'remote.origin.url'])).thenReturn({
             stdout: 'http://github.com/testOrg/testRepo.git'
```

Comment:
This is what I mean by an https:// URL. It has no colon

Reply:
I don't see a world where if this would have reared its head, it has been lying in wait all this time

https://github.com/codecov/codecov-bash/blob/master/codecov#L667